pyproject.toml

```diff
@@ -40,7 +40,7 @@ aws = [
 
 azure = [
     "azure-identity ~= 1.9.0",
-    "azure-mgmt-compute ~= 26.1.0",
+    "azure-mgmt-compute ~= 28.0.0",
```

Comment:
make this PR minimal to upgrading and fixing related places. If you want to include other small changes in the same PR, put them into separated commits.

Reply:
All the changes as part of this PR are flake8 error fixes. There are no extra changes included in this.